extensions/ruff/extension.toml

```diff
@@ -4,7 +4,7 @@ description = "Support for Ruff, the Python linter and formatter"
 version = "0.1.0"
 schema_version = 1
 authors = []
-repository = "https://github.com/zed-industries/zed"
+repository = "https://github.com/astral-sh/ruff"
```

Comment:
Actually this is used to specify where the repo lives for the extension and it actually is correct: https://github.com/zed-industries/zed/tree/main/extensions/ruff.